src/Tribe/API.php

```diff
@@ -454,11 +454,25 @@ protected static function prepare_event_date_meta( $event_id, $data ) {
 
 			// Additionally store datetimes in UTC
 			if ( empty( $data['EventStartDateUTC'] ) ) {
-				$data['EventStartDateUTC'] = Tribe__Events__Timezones::to_utc( $data['EventStartDate'], $data['EventTimezone'] );
+				if ( Tribe__Events__Timezones::is_utc_offset( $data['EventTimezone'] ) ) {
```

Comment:
Collect the result of `Tribe__Events__Timezones::is_utc_offset( $data['EventTimezone'] )` in a variable to avoid running the same logic twice for a small gain.